lib/web/apiserver.go

```diff
@@ -92,6 +93,14 @@ import (
 const (
 	// SSOLoginFailureMessage is a generic error message to avoid disclosing sensitive SSO failure messages.
 	SSOLoginFailureMessage = "Failed to login. Please check Teleport's log for more details."
+
+	assistantTokensPerHour = 140
+	// assistantLimiterRate is the rate (in tokens per second)
+	// at which tokens for the assistant rate limiter are replenished
+	assistantLimiterRate = rate.Limit(assistantTokensPerHour / float64(time.Hour/time.Second))
```

Comment:
nit:

```suggestion
	assistantLimiterRate = rate.Limit(assistantTokensPerHour / time.Hour.Seconds())
```

Reply:
`time.Hour.Seconds()` sadly can not be used in a `const` definition